roles/xteve/tasks/subtasks/migration.yml

```diff
@@ -0,0 +1,28 @@
+#########################################################################
+# Title:         Community: xTeve | Migration Tasks                     #
+# Author(s):     Satz                                                   #
+# URL:           https://github.com/Cloudbox/Community                  #
+# --                                                                    #
+#         Part of the Cloudbox project: https://cloudbox.works          #
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+- name: Check folder exists
+  stat:
+    path: /opt/xteve
+  register: folder
+
+- name: Check subfolder exists
+  stat:
+    path: /opt/ombi/app
```

Comment:
Correction to reference of Ombi please.

Reply:
Thanks for pointing out migz. Corrected the same.

Reply:
Thanks for that